docs/_static/styles.css

```diff
@@ -0,0 +1,565 @@
+/* custom styles */
```

Comment:
Did you write this? Or did it come from somewhere else?

Reply:
I wrote everything up to the `/* AUTODOC */` bit (I just wanted to use another theme's autodoc styles).  If you don't know how to use chrome dev tools to find exactly the CSS you need to change I can show you!

Reply:
also... the autodoc bit would be _way_ shorter if I used a sass preprocessor.  but I don't plan to change it much